### ISSUE TITLE
disableConsoleOutput breaks code with QuickJS

### ISSUE THREAD

**Issue by ddzobov** [javascript-obfuscator/javascript-obfuscator#803]
`{"type":"error","description":"TypeError: 'console' is read-only","stack":"TypeError: 'console' is read-only\n    at <anonymous> (/loader.js:1)\n    at apply (native)\n    at <anonymous> (/loader.js:1)\n    at <anonymous> (/loader.js:1)\n  js:1)\n    at <anonymous> (/loader.js:1)\n    at <anonymous> (/loader.js:1)\n    at <eval> (/loader.js:1)","fileName":"/loader.js","lineNumber":1,"columnNumber":1}`

Maybe you can add something like removeConsoleOutput, which will remove all console.* calls from code?

**Comment by sanex3339** [comment]
Just disable this option

**Comment by ddzobov** [comment]
> Just disable this option

But with console output some debug info can be leaked, it is not good solution for me

**Comment by da411d** [comment]
I agree. For example, in webpack optimizer also removes `console.*` and `debugger` calls.
